datadog_cluster_agent/README.md

```diff
@@ -0,0 +1,59 @@
+# Datadog Cluster Agent Integration
+
+## Overview
+
+This check monitors the [Datadog Cluster Agent][1].
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+The datadog_cluster_agent check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `datadog_cluster_agent.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your datadog_cluster_agent performance data. See the [sample datadog_cluster_agent.d/conf.yaml][3] for all available configuration options.
+
+Only the following parameter is required:
+
+| Parameter        | Description                                                                                                                                                                                                                                                              |
+| ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `prometheus_url` | The URL where the Datadog Cluster Agent metrics are exposed by Prometheus (must be unique).                                                                                                                                                                              |
+
+This integration is based on OpenMetrics, so for more configurations, see [Prometheus and OpenMetrics Metrics Collection][8].
+
+2. [Restart the Agent][4].
+
+### Validation
+
+[Run the Agent's status subcommand][5] and look for `datadog_cluster_agent` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][6] for a list of metrics provided by this check.
+
+### Service Checks
+
+datadog_cluster_agent does not include any service checks.
```

Comment:
The openmetrics base check does submit a service check:
https://github.com/DataDog/integrations-core/blob/d2b02ee053c737f32edc032ce528a6174e3e2eba/datadog_checks_base/datadog_checks/base/checks/openmetrics/mixins.py#L709-L710